build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+import org.jlleitschuh.gradle.ktlint.tasks.BaseKtLintCheckTask
+
+buildscript {
+
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        // mostly migrated to new style plugin declarations, but some cross plugin interaction still requires this
+        classpath("com.android.tools.build:gradle:${project.extra["kaluga.androidGradlePluginVersion"]}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${project.extra["kaluga.kotlinVersion"]}")
+    }
+}
+
+plugins {
+    id("kaluga-library-components")
+    id("convention.publication")
+    id("org.jlleitschuh.gradle.ktlint")
+    id("org.jlleitschuh.gradle.ktlint-idea")
+    id("org.jetbrains.kotlin.multiplatform") apply false
+    id("org.jetbrains.kotlinx.binary-compatibility-validator")
+}
+
+// TODO: To be removed once we will migrate to kotlin version 1.6.20
+// https://youtrack.jetbrains.com/issue/KT-49109#focus=Comments-27-5667134.0-0
+rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin>() {
+    rootProject.extensions.getByType(org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension::class.java).nodeVersion = "16.13.2"
```

Comment:
Up for removal now?

Reply:
Sorry, fixed upstream. Removed from this PR as well